docs/kubernetes/ingress-usage.md

```diff
@@ -453,7 +453,7 @@ Set a Cookie in the response path of your clients.
 
 ### Authorization
 
-Our [autnetication and authorization tutorial](../tutorials/auth/)
+Our [autnetication and authorization tutorial](../../tutorials/auth/)
```

Comment:
`../tutorials/auth.md` would produce the right link no matter what path the docs are served under.

Reply:
we should redo the links ourselves

Reply:
Thanks for the PR we will fix up all links at a later point since there's probably more than just this one. Your change makes it work so it's better than not having it :)